Makefile

```diff
@@ -13,7 +13,7 @@ objects		= $(HTTP_PARSER_OBJ) \
 		             $(wildcard $(SOURCE_DIR)/*.c))
 
 CPPFLAGS	+= $(PYTHON_INCLUDE) -I . -I $(SOURCE_DIR) -I $(HTTP_PARSER_DIR)
-CFLAGS		+= $(FEATURES) -std=c99 -fno-strict-aliasing -fcommon -fPIC -Wall
+CFLAGS		+= $(FEATURES) -std=c11 -fno-strict-aliasing -fcommon -fPIC -Wall
```

Comment:
I have blindly removed StringIO from py2 implementation and things still works. Please help to review something must be missing ... :-)